pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml

```diff
@@ -1,9 +1,27 @@
-%link{href: "/css/dance-landing/dance.css", type: "text/css", rel: "stylesheet"}
+- if request.language == 'en'
+  %link{href: "/css/dance-landing/dance.css", type: "text/css", rel: "stylesheet"}
 
-.sponsor-logo-with-text
-  = I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
-  .sponsor-logo-image
-    %img{src:"/images/fit-250/AFE-dark-logo.png"}
+  .sponsor-logo-with-text{style: "justify-content: center;"}
+    = I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
+    .sponsor-logo-image
+      %img{src:"/images/fit-250/AFE-dark-logo.png"}
 
-.sponsor-subtext-description
-  != I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_description_with_link_markdown, markdown: :inline, amazon_future_engineer_link: 'https://www.amazonfutureengineer.com')
+  .sponsor-subtext-description
+    != I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_description_with_link_markdown, markdown: :inline, amazon_future_engineer_link: 'https://www.amazonfutureengineer.com')
+
+- else
+
+  %link{href: "/css/full-resource-block.css", rel: "stylesheet"}
+  %link{href: "/css/tools.css", rel: "stylesheet"}
+
+  .sponsor-logo-with-text
+    = I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
+    .sponsor-logo-image
+      %img{src:"/images/fit-250/AFE-dark-logo.png"}
+
+  .full-resource-block{style: "margin-top: 20px;"}
+    %a{href: "https://www.amazonfutureengineer.com", target: "_blank"}
```

Comment:
tiny NIT: now that we're using this url twice, i'd consider putting it in a constant.

Reply:
added! thanks!